### PR TITLE
gateway: user can run simulate sample

### PR DESCRIPTION
**Pull request by yuwzho**
@yaweiw @zikalino

### CODE REVIEW

File: gateway.js [hunk 1]
```diff
@@ -42,11 +46,15 @@ function initTasks(gulp, options) {
...
     var fileList = [
       '.ble_gateway.json',
+      '.simulate_device_cloud_upload.json',
       'discover-sensortag.js',
       'test-connectivity.js',
       'deploy.js',
-      'run.js',
-      'lib/bleconfig.js',
+      'run-ble-sample.js',
+      'run-simulate-sample.js',
```
Comment (yaweiw) [inline gateway.js]: run-simudev-sample.js?

File: gateway.js [hunk 2]
```diff
@@ -42,11 +46,15 @@ function initTasks(gulp, options) {
...
+      'lib/ble-config.js',
+      'lib/gateway-config.js',
+      'lib/simulate-config.js',
```
Comment (yaweiw) [inline gateway.js]: simudev-config.js

**Review by yaweiw — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "gulp-common",
-  "version": "v0.11.2",
+  "version": "v0.11.3",
```
Comment (xscript) [inline package.json]: Current version should be v0.11.5. Two versions are skipped in previous PRs.

File: gateway.js (hunk 1 above)
Comment (yuwzho) [reply gateway.js]: yes, renamed

File: gateway.js (hunk 2 above)
Comment (yuwzho) [reply gateway.js]: renamed

File: package.json (hunk 3 above)
Comment (yuwzho) [reply package.json]: yes, it is. But I wonder, why the version in package.json is out of date...

Comment (xscript) [reply package.json]: Yaohai commited new changes before I send the email.

**Review by xscript — APPROVED** [review]

**Pull request merged**